share/smack/lib/smack.rs

```diff
@@ -0,0 +1,14 @@
+extern {fn __VERIFIER_assert(x: i32);}
+
+#[macro_export]
+macro_rules! assert {
+    ( $cond:expr ) => ( unsafe {__VERIFIER_assert($cond as i32);} )
```

Comment:
Let's use indentation of two spaces. I hope that is fine for Rust.

Reply:
Done